server/src/product/product.service.ts

```diff
@@ -47,9 +47,14 @@ export class ProductService {
     return product;
   }
 
-  async findAll(userId: number) {
+  async findAll(userId: number, isSale?: boolean) {
+    let whereQuery = '';
+    if (isSale) {
+      whereQuery = `author.id=${userId}`;
+    }
     const products = await Product.getProductQuery(userId)
       .select(['product', 'author', 'thumbnail', 'isLiked', 'region'])
+      .where(whereQuery)
```

Comment:
whereQuey 부분 삼항 연산자 쓰는 건 어떤가요?
```suggestion
      .where(isSale ? `author.id=${userId}` : '')
```

Reply:
넵 수정했습니다!!